core/src/main/java/tc/oc/pgm/api/map/Phase.java

```diff
@@ -0,0 +1,18 @@
+package tc.oc.pgm.api.map;
+
+import static net.kyori.adventure.text.Component.translatable;
+
+import net.kyori.adventure.text.Component;
+
+public enum Phase {
+  PRODUCTION,
+  DEVELOPMENT;
+
+  public Component toComponent() {
+    return translatable("map.phase." + this.name().toLowerCase());
+  }
+
+  public static Phase of(String query) {
+    return query.startsWith("dev") ? DEVELOPMENT : PRODUCTION;
```

Comment:
A bit of a too lenient check, phase "kdnfgojkdfjngjidfn" would match as production.

I'd iterate over Phase.values(), and check for name().toLowerCase().startsWith(query), if none match return a null and make the command show a proper exception about it